packages/financial-templates-lib/src/price-feed/FallBackPriceFeed.js

```diff
@@ -68,6 +68,11 @@ class FallBackPriceFeed extends PriceFeedInterface {
     }
   }
 
+  // Return the longest lookback within all the fallback feeds.
```

Comment:
Shouldn't we match what the [medianizer](https://github.com/UMAprotocol/protocol/blob/24073fcd91cf959a7dd1e8156420bf32a7ea5080/packages/financial-templates-lib/src/price-feed/MedianizerPriceFeed.js#L111-L117) does and take the min instead of the max?

Reply:
does shortest make the most sence? my reasoning is if any of the feeds within the FallBackFeed have a longer duration then we should take the longest one to maximise the changes of overlapping with the desired timestamp.

Reply:
Ah, good point. Agreed.